lib/krikri/logger.rb

```diff
@@ -0,0 +1,38 @@
+module Krikri
+  ##
+  # An application-wide tagged logger
+  # 
+  # @example with default logger
+  #   Krikri::Logger.log :info, 'message'
+  #
+  # @example with custom logger
+  #   logger = Logger.new(my_logger)
+  #   logger.log :info, 'message'
+  class Logger
+    ##
+    # @param [ActiveSupport::TaggedLogging] logger
+    def initialize(logger = ActiveSupport::TaggedLogging.new(Rails.logger))
```

Comment:
I understand the intent here (i.e., to be able to potentially swap in other tagged logging implementations), but is the assumption that anything would expose a similar interface as `ActiveSupport::TaggedLogging`, based on what's on [line 34](https://github.com/dpla/KriKri/pull/191/files#diff-31e82b2b381da2aeda2c21013a337c1dR34) below?